.gitlab-ci.yml

```diff
@@ -0,0 +1,36 @@
+include:
+  - project: 'analytics/artificial-intelligence/ai-platform/aip-infrastructure/ci-templates/ci-cd-template'
+    ref: &include_ref 'alexla/AIP-4600'  # TODO AIP-4600 Replace with v2
```

Comment:
`# TODO AIP-4600 Replace with v2` -> the end goal is to pin to `master`, right? Or would it be `feature/zg`?

Reply:
No the idea would be to replace the `ref` with `v2`. We now automatically `git tag` the `ci-cd-template` repos themselves when we make changes (see [this PR](https://gitlab.zgtools.net/analytics/artificial-intelligence/ai-platform/aip-infrastructure/ci-templates/ci-cd-template/-/merge_requests/105)). 

In [this PR](https://gitlab.zgtools.net/analytics/artificial-intelligence/ai-platform/aip-infrastructure/ci-templates/ci-cd-template/-/merge_requests/106) (that is coupled to the `v3.9` upgrade work), this bumps the version of the `ci-cd-template` publishing to `v2.0` and we use best effort tag overrides (not great but the least worst option Gitlab offers for any kind of semi-automated delivery mechanism, no such thing as lockfiles in Gitlab yet) to distribute updates to the CICD templates.